OpenChange/RTFHandler.m

```diff
@@ -378,26 +407,149 @@ - (RTFColorDef *) colorDefAtIndex: (unsigned int) theIndex
 //
 @implementation RTFHandler
 
-- (id) initWithData: (NSData *) theData
+static NSMapTable *_charsets = nil; 
+static NSMapTable *_cws = nil;
+typedef enum {
+  CW_UNKNOWN = 0,
+  CW_ANSICPG,
+  CW_B,
+  CW_CF,
+  CW_COLORTBL,
+  CW_F,
+  CW_FONTTBL,
+  CW_I,
+  CW_PAR,
+  CW_PICT,
+  CW_SOFTLINE,
+  CW_STRIKE,
+  CW_STYLESHEET,
+  CW_TAB,
+  CW_U,
+  CW_UL,
+  CW_ULNONE
+} CW_ID;
```

Comment:
Why do use UPPER_CASE for a type definition? Shouldn't it be `CwId` or `CwIdType`?